phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexVerificationOutputRow.java

```diff
@@ -0,0 +1,221 @@
+        this.dataTableName = dataTableName;
+        this.indexTableName = indexTableName;
+        this.scanMaxTimestamp = scanMaxTimestamp;
+        this.dataTableRowKey = dataTableRowKey;
+        this.indexTableRowKey = indexTableRowKey;
+        this.dataTableRowTimestamp = dataTableRowTimestamp;
+        this.indexTableRowTimestamp = indexTableRowTimestamp;
+        this.errorMessage = errorMessage;
+        this.expectedValue = expectedValue;
+        this.actualValue = actualValue;
+        this.phaseValue = phaseValue;
+    }
+
+    public String getDataTableName() {
+        return dataTableName;
+    }
+
+    public String getIndexTableName() {
+        return indexTableName;
+    }
+
+    public Long getScanMaxTimestamp() {
+        return scanMaxTimestamp;
+    }
+
+    public byte[] getIndexTableRowKey() {
+        return indexTableRowKey;
+    }
+
+    public long getIndexTableRowTimestamp() {
+        return indexTableRowTimestamp;
+    }
+
+    public String getErrorMessage() {
+        return errorMessage;
+    }
+
+    public byte[] getExpectedValue() {
+        return expectedValue;
+    }
+
+    public byte[] getActualValue() {
+        return actualValue;
+    }
+
+    public byte[] getPhaseValue() {
+        return phaseValue;
+    }
+
+    public byte[] getDataTableRowKey() {
+        return dataTableRowKey;
+    }
+
+    public Long getDataTableRowTimestamp() {
+        return dataTableRowTimestamp;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (o == null ) {
+            return false;
+        }
+        if (!(o instanceof  IndexVerificationOutputRow)) {
+            return false;
+        }
+        IndexVerificationOutputRow otherRow = (IndexVerificationOutputRow) o;
+
+        return Objects.equals(dataTableName, otherRow.getDataTableName()) &&
+            Objects.equals(indexTableName, otherRow.getIndexTableName()) &&
+            Objects.equals(scanMaxTimestamp, otherRow.getScanMaxTimestamp()) &&
+            Arrays.equals(dataTableRowKey, otherRow.getDataTableRowKey()) &&
+            Arrays.equals(indexTableRowKey, otherRow.getIndexTableRowKey()) &&
+            Objects.equals(dataTableRowTimestamp, otherRow.getDataTableRowTimestamp()) &&
+            Objects.equals(indexTableRowTimestamp, otherRow.getIndexTableRowTimestamp()) &&
+            Objects.equals(errorMessage, otherRow.getErrorMessage()) &&
+            Arrays.equals(expectedValue, otherRow.getExpectedValue()) &&
+            Arrays.equals(actualValue, otherRow.getActualValue()) &&
+            Arrays.equals(phaseValue, otherRow.getPhaseValue());
+    }
+
+    @Override
+    public int hashCode(){
+        return Objects.hashCode(scanMaxTimestamp) ^ Objects.hashCode(indexTableName) ^
+            Arrays.hashCode(dataTableRowKey);
+    }
+
+    @Override
+    public String toString() {
+        StringBuilder sb = new StringBuilder();
+        sb.append("DataTableName: ").append(dataTableName).append(",");
```

Comment:
Why not reuse the same strings from table values? like DTName, ITName. Would be easier to track and reuse.

Reply:
Good idea. Done.